recipes/open62541/all/conanfile.py

```diff
@@ -0,0 +1,300 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import glob
+import shutil
+import yaml
+
+
+
+class Open62541Conan(ConanFile):
+    name = "open62541"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Open source implementation of OPC UA (OPC Unified Architecture) aka IEC 62541 licensed under Mozilla Public License v2.0"
+    topics = ("conan", "opcua", "iec62541")
+    homepage = "http://open62541.org"
+    license = "MPL-2.0"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    exports = "submoduledata.yml"
```

Comment:
This should go in the `exports_sources`, it is only needed if we are going to build the package.